exercises/go-2-a/.meta/start_1_example.go

```diff
@@ -0,0 +1,50 @@
+package go2a
+
+import (
+	"fmt"
+	"strconv"
+)
+
+func split(amount int) (gross int, score int, dozen int, rest int) {
+	gross = amount / 144
+	amount %= 144
+
+	score = amount / 20
+	amount %= 20
+
+	return gross, score, amount / 12, amount % 12
+}
+
+func concat(description string, str string, remainder int) string {
+	if description != "" && remainder > 0 {
+		return description + ", " + str
+	} else if description != "" {
+		return description + " and " + str
+	} else {
```

Comment:
no else / else if

Reply:
Yes, could be improved. If there is a return in an `if`, there is no need for `else (if)`.

Note: This file is an example on how things _could_ be solved. As far as I understand this will not go out to the participant of the study, whereas the `start_X.go` files will. That is why they need to be clean, the `_example.go` are not so important.